CONTRIBUTING.md

```diff
@@ -186,7 +186,7 @@ As languages differ quite a lot, here is a set of captures available to you when
 @type.definition ; identifiers in type definitions (e.g. `typedef <type> <identifier>` in C)
 @type.qualifier  ; type qualifiers (e.g. `const`)
 
-@storageclass    ; modifiers that affect storage in memory or life-time
+@keyword.storage    ; modifiers that affect storage in memory or life-time
 @attribute       ; attribute annotations (e.g. Python decorators)
 @field           ; object and struct fields
 @property        ; similar to `@field`
```

Comment:
`@field` is changed to `@variable.member`, from what I can see. The doc should also be updated

Reply:
How should we document `@property` then?

Reply:
Very much unsure. `@property` usage is all over the place right now.

Reply:
That's not the only one...

Reply:
We will need community support to reliably convert `@property` into actually meaningful(relatable) captures ... So a backlog type of stuff I feel.

Reply:
I think I may have been too optimistic here; it's not enough to simply rename a bunch of captures, we need to also deprecate some and push for changes on every PR. Those won't be as breaking so can indeed be done down the line, but it would be good to get the ball rolling here and identify the ones we want to see the back of.

Reply:
This would also mean to **separate** from tree-sitter upstream queries, which goes against the intention of the renaming. As upstream also sometime uses `@property` incorrectly.

Reply:
Yes, that's what I suspect.

Reply:
(It's always the same queries that are fishy: re2c, chatito, capnp, git_commit, prompql, ini, ...)

Reply:
`@property` should be used for css properties and keys in data/config languages (e.g. json/toml/etc.)